src/app/euro.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DecimalPipe } from '@angular/common';
+
+@Pipe({name: 'euro'})
+export class EuroPipe extends DecimalPipe implements PipeTransform {
```

Comment:
at first i used currencypipe, but it doesn't allow me to customize much. in the end it was easiear to extend decimalpipe to add spaces and number sign